admin/server/organizations.go

```diff
@@ -844,6 +876,122 @@ func (s *Server) SudoUpdateOrganizationQuotas(ctx context.Context, req *adminv1.
 	}, nil
 }
 
+func (s *Server) SudoUpdateOrganizationBilling(ctx context.Context, req *adminv1.SudoUpdateOrganizationBillingRequest) (*adminv1.SudoUpdateOrganizationBillingResponse, error) {
```

Comment:
I have split this into two APIs - one sudo api for just assigning a customer id to the rill org, will be useful in case customer is already created or manually created in Orb. Second is for self serve plan change, this will be used for upgrading plans by customers, currently there is no check but that would require having a billing method.